packages/server/package.json

```diff
@@ -14,6 +14,12 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
-    "@0auth/message": "^0.0.1"
+    "@0auth/message": "^0.0.1",
+    "crypto-js": "^4.0.0",
+    "elliptic": "^6.5.3"
+  },
+  "devDependencies": {
+    "@types/crypto-js": "^3.1.47",
+    "@types/elliptic": "^6.4.12"
```

Comment:
Libraries for ECDSA, EdDSA, and SHA256 cryptographic algorithms.
Also I added dev dependency for use in typescript.